wizeline/remix-project-lab
# Project members and disciplines field already working

# What does this PR do?


In this PR, the project members field is already bringing the profiles from server and sending the data correctly, also the disciplines field is already finished.
# Where should the reviewer start?


`app/routes/projects/create.tsx`
# How should this be manually tested?



1. Enter the required inputs and some data to the project members and disciplines input and submit.
2. See the data in the project details page.

# Any background context you want to provide?



# What are the relevant tickets?



# Screenshots


<img width="915" alt="Screenshot 2022-12-06 at 14 26 23" src="https://user-images.githubusercontent.com/62604183/205927470-b5558224-b4e5-4062-96a6-3f7b207bc41e.png">

<img width="427" alt="Screenshot 2022-12-06 at 1 27 55" src="https://user-images.githubusercontent.com/62604183/205778303-2b42919d-bc50-4a07-a665-c9a8697d8177.png">


# Questions



# Checklist
